tests/worker/topics/test_rabbitmq_topic.py

```diff
@@ -189,3 +190,29 @@ async def test_rabbitmq_topic_serialization_error(
         await topic.publish(
             TopicMessage(id=MessageId("0"), args={"n": datetime.now()}), wait=True
         )
+
+
+@pytest.mark.asyncio
+async def test_rabbitmq_topic_expiring_message(
+    rabbitmq_topic_maker: t.Callable[..., Awaitable[RabbitMQTopic]],
+    frozen_time: FreezeTime,
+) -> None:
+    topic = await rabbitmq_topic_maker(RabbitMQTopic)
+
+    message = TopicMessage(id=MessageId("0"), args={"n": b"1"}, expire_after=None)
+    messages = [
+        TopicMessage(
+            id=MessageId("1"), args={"n": b"2"}, expire_after=timedelta(seconds=1)
```

Comment:
you can just set an expiration of 0s? no sleep needed.

Reply:
I'm surprised it even work

Reply:
the test is skip I am not how to run rabbitmq tests.

Reply:
I can try 0 sec :)